tests/static/js/formatters.js

```diff
@@ -62,10 +71,13 @@ describe('Formatters', () => {
       const price = Formatters.price(priceField, 'en');
       expect(price).toEqual('$100.99');
     });
+
     it('Formats a price in USD with no provided locale', () => {
+      document.documentElement.lang = 'en';
```

Comment:
does the documentElement.lang state need to be cleared after each test?

Reply:
document.documentElement.lang takes after the value set to it in the previous 'it' function. It doesn't affect our tests with the way we set them up now, but I will clear it out to avoid confusion if we do update these tests later